airflow-core/src/airflow/config_templates/airflow_local_settings.py

```diff
@@ -71,22 +71,22 @@
         },
     },
     "filters": {
-        "mask_secrets": {
-            "()": "airflow.sdk.execution_time.secrets_masker.SecretsMasker",
+        "mask_secrets_core": {
+            "()": "airflow._shared.secrets_masker.secrets_masker._secrets_masker",
```

Comment:
Woah! that's so repeititve: `.secrets_masker.secrets_masker._secrets_masker`

Reply:
Yikes, yeah thats not needed anymore. Updating it to `airflow._shared.secrets_masker._secrets_masker`

Reply:
> Any concerns about back compat?

I do not think it should be, because we are updating the name of the mask now, although maybe adding a backcompat shim for the old path would be good?

Reply:
It could be beneficial for users who copy the airflow_local_settings to lets say configmaps etc